.env.example

```diff
@@ -42,6 +42,15 @@ MAIL_FROM_NAME=
 MAILGUN_DOMAIN=
 MAILGUN_SECRET=
 
+# Slack Webhook Endpoint path to be appended to https://hooks.slack.com/services/.
+SLACK_ENDPOINT=
+
+# Default slack channel for the bot to post in.
+SLACK_CHANNEL=
+
+# Default username to post as
+SLACK_USERNAME=
```

Comment:
I, personally, feel like it would be a bit overkill to have a comment that a `SLACK_USERNAME` variable relates to the username that the bot would post as? (Surely it should be intuitive?) 
With regards to limiting the scope by adding the `_BOT_`, that variable by definition is only used by our slack 'bot' for the purpose of notification. I could also only ever see it being used as such. We would never need to use core's hook into slack to post as an actual member/person, or for it to be used by a 'human'.

I do, however, agree that we could add a comment to each block, but it would need to be concise, as primarily we want the user to look into the contributing document (which can be updated easier, with more room for fancy formatting) for help with initial configuration. The names of the variables should (hopefully) already provide some context as to their purpose.
Feel free to add a commit updating the contributing document to detail the new variables.

Regarding the CONTRIBUTING.md, it's there to help with that initial configuration, just like any other software would come with a manual, and it is there to explain the in's and outs in detail.